src/deavinaSJun20/TestIntSuminString.java

```diff
@@ -0,0 +1,19 @@
+package deavinaSJun20;
+
+public class TestIntSuminString {
+
+	public static void main(String[] args) {
+		String input = "J7yu9y8h1h8j4b7j3jjb6";
+		new TestIntSuminString().calculateSumOfDigitsInString(input);
+	}
+
+	void calculateSumOfDigitsInString(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				sum = sum + Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));